qgreenland/config/layers.yml

```diff
@@ -4375,7 +4375,7 @@
 # TODO: better id, title, description for these layers
 - &future_ice_sheet_coverage
   id: future_ice_sheet_coverage_rcp_26
-  title: 'Future (year 3007) ice sheet coverage - RCP 2.6 scenario (1.8km)'
+  title: 'Future ice sheet coverage - RCP 2.6 scenario for the year 3007 (1.8km)'
```

Comment:
Nitpick: We could change "for the year 3007" to "for year 3007" or just "for 3007" -- I think even the last one would be clear to our target audience.